system/HTTP/CLIRequest.php

```diff
@@ -214,4 +215,12 @@ public function isCLI(): bool
     {
         return true;
     }
+
+    /**
+     * Gets the current locale.
+     */
+    public function getLocale(): string
```

Comment:
Would it make sense to put this on Request (or RequestTrait) so any request has it? IncomingRequest can still use it's negotiable version as an extension.

Reply:
IncommingRequest has locale, but I don't think CLIRequest should have locale.

Okay, I was wrong. This method should not be added.